pypuck/pypuck.py

```diff
@@ -77,22 +78,21 @@ def player_stats(start_date=None, end_date=None):
     return pd.DataFrame(page.json()['data'])
 
 
-def attendance(regular=True, playoffs=True, season=None):
+def attendance(regular=True, playoffs=True, start_season=None, end_season=None):
     """
     Query to the Attendance API to get the NHL's seasonal and playoff attendance numbers.
        
```

Comment:
@chuusan could you add a little more description to this? For example, query the NHL attendance numbers from __________ to ____________ from the NHL records API. The attendance represents yearly attendance numbers for all teams combined.

The user can specify to return either the regular season attendance numbers, playoff attendance numbers, or both. The function will display a chart showing the attendance over the specified time period.

Reply:
Sure.